server/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+## [0.31.0] - 2023-05-30
+
+- **BREAKING CHANGES (VQB)**: Now we clean filter steps containing `__VOID__` as values in conditions while rendering a pipeline.
```

Comment:
Oh no I thought this has been removed. I believe this should be undone before making a release

Reply:
you mean, it's better to make two differents release for those changes ?
Thus a revert PR for that ___VOID ___ feature ?

Reply:
No, I mean that render should **not** take care of the `__VOID__` case. A method called `render` should render, that's it. If a user needs to remove `__VOID__` values, they can use the newly function for that

Reply:
So this line should be changed : https://github.com/ToucanToco/weaverbird/blob/master/server/src/weaverbird/pipeline/pipeline.py#L269 (because the PR is already merged)

i thought we were aware of the fact that the new render was about to remove that. @fspot any thoughts?

Reply:
yes the PR is merged, forgot to re-check that the remarks on this thread had been taken into account before merging https://github.com/ToucanToco/weaverbird/pull/1763#discussion_r1204039520

Anyway, here's a fix https://github.com/ToucanToco/weaverbird/pull/1786

Reply:
yes I'm aligned with https://github.com/ToucanToco/weaverbird/pull/1763#discussion_r1205143694

Reply:
Thanks for the update/PR, according to these new information, I updated the Changelog and the version naming here : https://github.com/ToucanToco/weaverbird/pull/1782/commits/4feccd711676e06d80470547df49409b3b63103f